tools/CouchDB_Import_Instruments.php

```diff
@@ -278,7 +288,7 @@ function updateCandidateDocs($Instruments)
      */
     function getInstruments()
```

Comment:
this function should take a lorisinstance as a parameter no?

Reply:
why? It has access to the class variable..

Reply:
line 325, you are passing it in arent you ?

Reply:
Oops.. just fixed line 325.